source/manual/gtm-change-process.html.md

```diff
@@ -0,0 +1,62 @@
+---
+owner_slack: "#govuk-frontenders"
+title: 'Google Tag Manager change process'
+section: Frontend
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+related_repos: [govuk_publishing_components]
+---
+
+GOV.UK has implemented Google Analytics 4 (GA4) using Google Tag Manager (GTM) to track user journeys through the site. This page describes the process of making changes to GTM to support improvements to our analytics.
+
+This process exists because some changes can increase the size of the JavaScript included from GTM in the public frontend of GOV.UK. We do not want to unnecessarily degrade the performance of GOV.UK.
+
+## Roles and responsibilities
+
+Performance analysts:
+
+- to make changes in GTM and share the change with a succinct explanation of what it does and why it's needed
+- to fully document the details of the change
+
+Developers:
+
+- to review and publish the change in GTM
+- to confirm that the changes won't break anything
+- to check what's going on is as expected
+
+## Process
+
+Analyst creates a change in the GTM web interface.
+
+- change is created in a new workspace
+- change must include a clear description
+- change can be checked in preview mode (optional)
+
+Analyst gets a second analyst to check the changes.
+
+Analyst raises a request to have the change approved.
+
+- can be raised in the `#govuk-ga4` slack channel
```

Comment:
I wonder if we can ask for the notification to be put into #user-experience-measurement-govuk, or maybe use a 'here' in #govuk-ga4 instead? I know I've missed some of these requests in the past.

Reply:
That's a good point, have changed the channel.